EIPS/eip-7916.md

```diff
@@ -38,30 +67,108 @@ ProgressiveByteList[3000, 4]
      \/
     ROOT
 
-V1: ByteVector[32]
-V2: ByteVector[128]
-V3: ByteVector[512]
-V4: ByteVector[2048]
-V5: ByteVector[512]
+V1: Vector[T, 1]
```

Comment:
Here the vector capacities are the same as the number of chunks in each subtree. This is only true if vector elements have a 32 byte size or are stored as `hash_tree_root`. Maybe an explanatory comment is enough here that this is an assumption and the vectors have fixed chunk count, not fixed element capacity.

Reply:
Thanks, yes, that's true. Have added a comment.